src/map/entities/baseentity.h

```diff
@@ -318,7 +318,7 @@ class CBaseEntity
 
     bool isRenamed; // tracks if the entity's name has been overidden. Defaults to false.
 
-    bool m_bReleaseTargIDOnDisappear;
+    bool m_bReleaseTargIDOnDisappear = false;
```

Comment:
@LoxleyXI Why did this get changed to initialization here? It's being initialized on the constructor for the class.